source/features/link-to-changelog-file.tsx

```diff
@@ -9,6 +9,7 @@ import features from '../feature-manager.js';
 import api from '../github-helpers/api.js';
 import {wrapAll} from '../helpers/dom-utils.js';
 import {buildRepoURL, getRepo} from '../github-helpers/index.js';
+import GetFilesOnRoot from './show-associated-branch-prs-on-fork.gql';
```

Comment:
Shouldn't this be

```suggestion
import GetFilesOnRoot from './link-to-changelog-file.gql';
```

?

Reply:
@fregante This was not fixed, right?

Reply:
Oops, no

Reply:
So now `link-to-changelog-file` is probably broken in the latest release?